docs/kits/Business Partner Kit/Documentation BPDM/page_use-cases.md

```diff
@@ -0,0 +1,2294 @@
+                "bpn": "BPNL00000006ME6Y",
+                "identifiers": [
+                    {
+                        "value": "63485ec63cd3b21131fb69ac",
+                        "type": {
+                            "technicalKey": "CDQID",
+                            "name": "CDQ Identifier",
+                            "url": ""
+                        },
+                        "issuingBody": {
+                            "technicalKey": "CDQ",
+                            "name": "CDQ AG",
+                            "url": ""
+                        },
+                        "status": {
+                            "technicalKey": "CDQ_IMPORTED",
+                            "name": "Imported from CDQ but not synchronized"
+                        }
+                    }
+                ],
+                "names": [],
+                "legalForm": null,
+                "status": null,
+                "profileClassifications": [],
+                "types": [],
+                "bankAccounts": [],
+                "roles": [],
+                "relations": [],
+                "currentness": "2022-10-14T15:00:06.325509Z"
+            }
+        },
+        {
+            "score": 0.0,
+            "legalEntity": {
+                "bpn": "BPNL00000006MF69",
+                "identifiers": [
+                    {
+                        "value": "63485ec63cd3b21131fb69ae",
+                        "type": {
+                            "technicalKey": "CDQID",
+                            "name": "CDQ Identifier",
+                            "url": ""
+                        },
+                        "issuingBody": {
+                            "technicalKey": "CDQ",
+                            "name": "CDQ AG",
+                            "url": ""
+                        },
+                        "status": {
+                            "technicalKey": "CDQ_IMPORTED",
+                            "name": "Imported from CDQ but not synchronized"
+                        }
+                    }
+                ],
+                "names": [],
+                "legalForm": null,
+                "status": null,
+                "profileClassifications": [],
+                "types": [],
+                "bankAccounts": [],
+                "roles": [],
+                "relations": [],
+                "currentness": "2022-10-14T15:00:06.325574Z"
+            }
+        }
+    ]
+}
+```
+
+### 1.3 Read out BPNL from API - Visualize BPNL for legal Entity
+
+Flow:
+
+* The user inputs a valid EU_VAT_ID_DE to the Legal Entity Controller.
+* The Legal Entity Controller sends an API call to the internal API with the EU_VAT_ID_DE.
+* The internal API returns information about the legal entity.
+* The Legal Entity Controller reads the BPNL (Business Partner Number List) from the API response.
+* The Legal Entity Controller displays the BPNL for the legal entity to the user e.g. "bpn": "BPNL00000006M6CI".
```

Comment:
What kind of use case is that? Feels like that it exceeds use case 1 in which you only want to request a legal entity by identifier number

Reply:
Yes, it is part of use case 1. May be i shall show you the page then it will be more clear or will omit this heading.